### PR TITLE
v2.0.4 preview

### PR DESCRIPTION
**Pull request by toots**
Co-authored-by: Martin Kirchgessner <martin.kirch@gmail.com>

### CODE REVIEW

**Comment by finnie2006** [comment]
Any eta on this? because liquidsoap is not working for me right now on arm

**Comment by toots** [comment]
> Any eta on this? because liquidsoap is not working for me right now on arm

I'm hoping this coming week. Does `2.0.4` work for you?

**Comment by finnie2006** [comment]
Not tried it yet, i am using azuracast on my raspberry pi, and the developer from azuracast said this, so i will wait for the new liquidsoap in azuracast
image

File: src/encoder/shine_encoder.ml [hunk 1]
```diff
@@ -44,7 +44,6 @@ let encoder shine =
   let encoded = Strings.Mutable.empty () in
   let encode frame start len =
     let b = AFrame.pcm frame in
-    let start = Frame.audio_of_main start in
```
[the remark this answers is not in the document]
Comment (toots) [reply src/encoder/shine_encoder.ml]: Good catch, thank you.

**Pull request closed without merging**